README.md

```diff
@@ -73,20 +73,25 @@ To install prerequisite packages on the Platform9 Express control host, run the
 ## Configuration Inventory (CLI Only)
 Platform9 Express uses Ansible to execute commands on the hosts to be taken under management.  In order to configure Ansible to run remote commands on the managed hosts, the Ansible Inventory file must be configured.  This file is located in /opt/pf9-express/inventory/hosts.
 
-NOTE: A sample template is installed in the previous command ("./pf9-express -s").
+NOTE: A sample template is installed in the previous command ("./pf9-express -s"). Breaking down the Inventory File to separate Openstack and Kubernetes below:
 
-## Sample Inventory File
+## Sample Inventory File Part 1 - Authentication Portion
+This is where you enter the credentials for your control host to log into the target VM hosts to be managed by the Platform9 control plane (through either a password or SSH key, comment out any password lines if using SSH authentication and vice versa as needed)
 ```
 ##
 ## Ansible Inventory
 ##
 [all]
 [all:vars]
 ansible_user=ubuntu
-ansible_sudo_pass=winterwonderland
-ansible_ssh_pass=winterwonderland
-#ansible_ssh_private_key_file=~/.ssh/id_rsa
+#ansible_sudo_pass=password01
+#ansible_ssh_pass=password02
+ansible_ssh_private_key_file=~/.ssh/id_rsa
+```
 
+## Sample Inventory File Part 2 - Openstack Portion
```

Comment:
Part 2 is Network and is applicable outside OpenStack. Please update to be a separate section and uppercase S in OpenStack.

Reply:
corrected